ruby-challenges.rb

```diff
@@ -10,6 +10,33 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+def even_odd num1
+if num1 % 2 == 0
```

Comment:
Logic looks great!  Consider how this method could be reused so you don't have to repeat it for each variable.